skimage/measure/_marching_cubes_lewiner.py

```diff
@@ -1,24 +1,55 @@
 import sys
 import base64
+import dis
+import inspect
 
 import numpy as np
 
 if sys.version_info >= (3, ):
     base64decode = base64.decodebytes
+    ordornot = lambda x:x
 else:
     base64decode = base64.decodestring
+    ordornot = ord
 
 from . import _marching_cubes_lewiner_luts as mcluts
 from . import _marching_cubes_lewiner_cy
+from .._shared.utils import skimage_deprecation, warn
 
 
-def marching_cubes_lewiner(volume, level=None, spacing=(1., 1., 1.),
-                           gradient_direction='descent', step_size=1,
-                           allow_degenerate=True, use_classic=False):
+def expected_output_args():
```

Comment:
Agreed, I'm impressed!

This may have other uses for similar deprecations in the future, so it might best fit in `skimage._shared`